tests/cypress/cypress/e2e/accessmatch/add-interests.cy.js

```diff
@@ -0,0 +1,22 @@
+/*  
+    This test is specifically focused on the Interests Page tested for an authenticated user.
+    This test checks for major functions like:
+    Page Title, 
+    Top Level Tag text,
+    Adding a Tag Functionality 
+
+    */
+
+describe("Authenticated user tests the Add Interests without adding an Affinity Group", () => {
+  it("Should test Add Interests Form for authenticated user", () => {
+    // login user with the "authenticated" role
+    cy.loginAs("pecan@pie.org", "Pecan");
+    cy.visit("/add-interest");
+    cy.contains("Add Interests");
+    cy.contains("ACCESS RPs");
+    cy.get(
+      ":nth-child(1) > :nth-child(2) > :nth-child(1) > .view-display-id-page_2 > .view-content > .add-interest-list > ul > :nth-child(1) > :nth-child(1) > span > .d-inline-flex > .d-none > .flag > .use-ajax"
+    ).click();
+    cy.contains("Remove");
+  });
```

Comment:
add interests is now under community persona, so this code looks pretty old.  i think a good test of adding interests is:

- on cp/add-interests as an auth user, click a few interests
- verify those ui element are changed (like 'selected' or whatever the change is)
- on the users persona page, verify they now have those interests
- back on add-interest, unselect those interests
- verify the ui changes
- on the cp page, verify they don't have those interests